eng/pipelines/vmr-internal-sync.yml

```diff
@@ -0,0 +1,33 @@
+pr: none
+
```

Comment:
Can you please do following to both of the sync pipelines:

1. Set `batch` to `false`
2. Add this to top-level
```
lockBehavior: sequential
```

Reply:
Or.. actually, let's wait with this.. I need to understand the docs better first